viewer/packages/cad-geometry-loaders/src/sector/SectorDownloadScheduler.ts

```diff
@@ -58,10 +71,43 @@ export class SectorDownloadScheduler {
     });
   }
 
+  private createDiscardedConsumedSector(sector: WantedSector): ConsumedSector {
+    return {
+      modelIdentifier: sector.modelIdentifier,
+      metadata: sector.metadata,
+      levelOfDetail: LevelOfDetail.Discarded,
+      group: undefined,
+      instancedMeshes: undefined
+    };
+  }
+
+  private abortLoadOfDiscardedSector(sector: WantedSector, sectorIdentifier: string): Promise<ConsumedSector> | null {
```

Comment:
I generally prefer undefined over null (what is used generally in rest of code base), but if there is a reason to prefer null here then we can keep it as is.